rpm_spec/manageiq.spec.in

```diff
@@ -87,6 +87,10 @@ cd %{_builddir}
 %{__cp} -r %{gemset_builddir}/vmdb %{buildroot}%{gemset_root}
 install -m644 %{gemset_builddir}/enable %{buildroot}%{gemset_root}
 
+#copy systemd unit files
+%{__cp} %{appliance_builddir}/build/systemd_units/* %{buildroot}/etc/systemd/system
+%{__cp} %{gemset_builddir}/bundler/gems/*/build/systemd_units/* %{buildroot}/etc/systemd/system/
```

Comment:
You always need to create a directory in %{buildroot}, so need to add:
`%{__mkdir} -p %{buildroot}%{_sysconfdir}/systemd/system`

Using macro is recommended over hard-coded path, so used `%{_sysconfdir}` instead of `/etc`